.travis.yml

```diff
@@ -35,7 +35,16 @@ jobs:
       name: "Deploy to Sonatype's snapshots repository"
       jdk: openjdk8
       if: type != pull_request AND env(SONATYPE_NEXUS_USERNAME) IS present
-      script: bash .travis.deploy.artifacts.sh
+      script:
+        - cd vertx-web-graphql/vertx-web-graphiql
+        - GENERATE_SOURCEMAP=false npm run build
+        - cd build
```

Comment:
Why not just do a `cp -Rf ... src/main/resources/io...` as part of the build on NPM and have it all in one place?

Reply:
Because there are other files that are not needed.

Reply:
Because for local testing it can be useful to run a production build. With the second part it would only work on *nix.